cirq/work/observable_measurement.py

```diff
@@ -153,7 +211,7 @@ def _pad_setting(
 
 
 def _aggregate_n_repetitions(next_chunk_repetitions: Set[int]) -> int:
-    """In the future, we will allow each accumulator to request a different number
+    """Each stopping criteria can request a different number
```

Comment:
IIUC, the rep counts passed to this function are all from the same stopping criteria, but for different measurement specs. This comment should reflect that.

Reply:
good point.